apps/landing/package.json

```diff
@@ -18,30 +18,30 @@
     "@emotion/server": "^11.4.0",
     "@emotion/styled": "^11.8.1",
     "@mui/icons-material": "^5.4.4",
-    "@mui/material": "^5.4.4",
+    "@mui/material": "^5.15.19",
     "@react-spring/web": "^9.3.1",
     "@sentry/nextjs": "^7.80.0",
-    "next": "12.1.5",
+    "next": "^14.2.3",
     "next-dark-mode": "^3.0.0",
-    "next-pwa": "^5.4.6",
-    "next-seo": "^5.5.0",
+    "next-pwa": "^5.6.0",
+    "next-seo": "^6.1.0",
     "nextjs-google-analytics": "^2.0.0",
-    "notistack": "^2.0.5",
+    "notistack": "2.0.8",
     "nprogress": "^0.2.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.3.1",
```

Comment:
I think we should have the version without `^` so we don't have surprise updates.

Also, there was a new version that kind of introduced a weird warning about `fetchPriority` props on a DOM element. [People are saying that in `18.2.0` is fine](https://github.com/vercel/next.js/issues/65161)

Reply:
ok, I'll try that. versions are locked anyway. but if it'll fix that warning it makes sense

Reply:
done